src/test/java/nitezh/ministock/domain/stocks.csv

```diff
@@ -0,0 +1,3 @@
+DF
```

Comment:
What is this file? Should it be committed to git?

Reply:
its for import tests while using an emulator.